anyway/views/news_flash/api.py

```diff
@@ -94,7 +95,7 @@ def news_flash_v2():
     if "id" in validated_query_params:
         return get_news_flash_by_id(validated_query_params["id"])
 
-    query = gen_news_flash_query_v2(validated_query_params)
+    query = gen_news_flash_query(db.session, validated_query_params)
```

Comment:
Regarding API versioning and function names.
I think it is not a good idea to change function names. If UI wants to move API version and stay with same API URL, then I think we should only change the end point definition to point to a new function.